### PR TITLE
rt: fix links to papers

### PR DESCRIPTION
**Pull request by corlewis**
This just cherry-picks two commits from master that should fix the links test.

### CODE REVIEW

**Comment by wom-bat** [comment]
This looks good to me.

File: tools/autocorres/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ abstracts the result to produce a result that is (hopefully)
 more pleasant to reason about.
 
   [1]: https://www.cl.cam.ac.uk/research/hvg/Isabelle/
-  [2]: https://ts.data61.csiro.au/software/TS/c-parser/
+  [2]: https://ts.data61.csiro.au/software/TS/c-parser.html
```
Comment (wom-bat) [inline tools/autocorres/README.md:19]: This one's wrong: the trailing slash is required.

Comment (wom-bat) [reply tools/autocorres/README.md:19]: Looks like there's an error in the web copying.  I'll fix it in the website.

**Comment by corlewis** [comment]
> 
> 
> This looks good to me.

Great! These commits fixed the test in master but were never merged into the rt branch so it's still been failing here.

File: tools/autocorres/README.md (hunk 1 above)
Comment (wom-bat) [reply tools/autocorres/README.md:19]: I've fixed this one upstream now.

**Review by wom-bat — CHANGES_REQUESTED** [review]

File: tools/autocorres/README.md (hunk 1 above)
Comment (corlewis) [reply tools/autocorres/README.md:19]: Thanks! I'll switch it back to `https://ts.data61.csiro.au/software/TS/c-parser/`

Comment (wom-bat) [reply tools/autocorres/README.md]: Cool.
With that should be OK to merge ...

**Review by wom-bat — APPROVED** [review]
LGTM now .

**Pull request merged**